Missing config.guess and config.sub

### Description
`config.guess` and `config.sub` are missing/not created for the install procedure documented on [README.Linux.md](https://github.com/OpenModelica/OpenModelica/blob/master/OMCompiler/README.Linux.md).

### Steps to Reproduce

Doing a fresh `git clone` then `autoconf` and `./configure` leads to an error:
```
$ MY_FORK=hkiel ; git clone https://openmodelica.org/git-readonly/OpenModelica.git --recursive && (cd OpenModelica && git remote set-url --push origin git@github.com:$MY_FORK/OpenModelica.git && git submodule foreach --recursive 'git remote set-url --push origin `git config --get remote.origin.url | sed s,^.*/,git@github.com:'$MY_FORK'/,`')$ cd OpenModelica
$ autoconf
configure.ac:95: warning: AC_OUTPUT should be used without arguments.
configure.ac:95: You should run autoupdate.
$ ./configure 
configure: error: cannot find required auxiliary files: config.guess config.sub
```

You can create them by hand by running `autoreconf --install`

### Expected Behavior

All files should be automatically created, or the installation procedure must be updated.

--- a/OMCompiler/README.Linux.md
+++ b/OMCompiler/README.Linux.md
@@ -37,7 +37,7 @@ sudo make install
 ```
 
 But first you need to install dependencies:
-- autoconf, automake, libtool, pkgconfig, g++, gfortran (pretty standard compilers)
+- autoconf, autoreconf, automake, libtool, pkgconfig, g++, gfortran (pretty standard compilers)
 - boost (optional, used with configure --with-cppruntime)
 - [clang](http://clang.llvm.org/), clang++ (optional, but *highly recommended*; if you use gcc instead, use gcc 4.4 or 4.9+, not 4.5-4.8 as they are very slow)
 - [cmake](http://www.cmake.org)
@@ -63,7 +63,7 @@ export PATH=${PATH}:/path/to/installed/mico/bin
 ## To Compile OpenModelica
 Run:
 ```bash
-autoconf
+autoreconf --install # Or autoconf if you have autoconf <=2.69
 # One of the following configure lines
 ./configure --with-omniORB=/path/to/omniORB (if you want omc to use omniORB corba)
 ./configure --with-CORBA=/path/to/mico (if you want omc to use mico corba)

--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ Many software packages are included inside the repositories.
 To get everything running, you will need a few extras:
 
 - C++11 compiler (if you want a GUI)
-- autoconf, automake, libtool, g++, gfortran (pretty standard compilers)
+- autoconf, autoreconf, automake, libtool, g++, gfortran (pretty standard compilers)
 - boost (used by omsimulator and cppruntime, configure --with-cppruntime)
 - [clang](http://clang.llvm.org/), clang++ (optional, but *highly recommended*)
 - [cmake](http://www.cmake.org)

